src/backend/src/services/organizations.services.ts

```diff
@@ -151,4 +151,27 @@ export default class OrganizationsService {
       exploreAsGuestImage: organization.exploreAsGuestImageId
     };
   }
+
+  /**
+   * Updates the application link for the given organization Id
+   * @param submitter the user who is setting the links
+   * @param organizationId organization Id of the organization
+   * @param newLink new application link to be updated
+   * @returns updated organization data
+   */
+  static async updateApplicationLink(submitter: User, organizationId: string, newLink: string) {
+    const organization = await prisma.organization.update({
+      where: { organizationId },
+      data: { applicationLink: newLink }
+    });
+
+    if (!(await userHasPermission(submitter.userId, organizationId, isAdmin)))
```

Comment:
Move this check above line 163 because we should be checking if the user is an admin before actually updating the organizationLink